src/js/components/Select/Select.js

```diff
@@ -231,7 +231,9 @@ const Select = forwardRef(
               {children}
             </SelectContainer>
           }
-          plain={plain}
+          // StyledDropButton needs to know if the border should be shown
+          callerPlain={plain}
```

Comment:
I'm probably missing something, but why do we need `callerPlain` if it performs the same as `plain`.

Reply:
Because StyledDropButton and Button both see the same properties. We want Button to do one thing but StyledDropButton to do something else.

Reply:
I think I got it. tricky.